app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```diff
@@ -152,6 +152,10 @@ $large-input-size: 50px;
       border-right: 0;
     }
 
+    .gem-c-search__input[type="search"].gem-c-search__suppressed-button{
```

Comment:
I'm not sure the `[type="search"]` is needed on this line, can you check?

Also, nitpick, should be a space before the `{`.

Reply:
I was trying to be consistent with the selector based on this comment: https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/stylesheets/govuk_publishing_components/components/_search.scss#L18

Any thoughts?